### PR TITLE
Ignitionrobotics to gazebosim

### PR DESCRIPTION
**Pull request by nkoenig**
## Summary

**DRAFT** I'm experimenting with changing the fuel servers.

This changes the default Fuel server from `fuel.ignitionrobotics.org` to `fuel.gazebosim.org`.

Additionally, requests to the `fuel.ignitionrobotics.org` server are changed to `fuel.gazebosim.org`. This is needed in order to support models that have dependencies, which are all using the `fuel.ignitionrobotics.org` server name. This should be okay since `fuel.ignitionrobotics.org` and `fuel.gazebosim.org` are the same. The downside is that users transitioning from Ignition to Gazebo will have two folders: 
1. `~/.ignition/fuel/fuel.ignitionrobotics.org`
2. `~/.ignition/fuel/fuel.gazebosim.org`.

Note that I'm not changing the `~/.ignition` base directory since that seemed like too major of a change for a released version of fuel tools.

## Checklist
- [ ] Signed all commits for DCO
- [ ] Added tests
- [ ] Updated documentation (as needed)
- [ ] Updated migration guide (as needed)
- [ ] Consider updating Python bindings (if the library has them)
- [ ] `codecheck` passed (See [contributing](https://ignitionrobotics.org/docs/all/contributing#contributing-code))
- [ ] All tests passed (See [test coverage](https://ignitionrobotics.org/docs/all/contributing#test-coverage))
- [ ] While waiting for a review on your PR, please help review [another open pull request](https://github.com/pulls?q=is%3Aopen+is%3Apr+user%3Aignitionrobotics+repo%3Aosrf%2Fsdformat+archived%3Afalse+) to support the maintainers

**Note to maintainers**: Remember to use **Squash-Merge** and edit the commit message to match the pull request summary while retaining `Signed-off-by` messages.

### CODE REVIEW

File: conf/config.yaml [hunk 1]
```diff
@@ -2,12 +2,12 @@
...
-    url: https://fuel.ignitionrobotics.org
+    name: gazebo
+    url: https://fuel.gazebosim.org
 
-  # -
-    # name: another_server
-    # url: https://myserver
+  -
+    name: ignition
+    url: https://fuel.ignition.org
```
Comment (chapulina) [inline conf/config.yaml:10]:
```suggestion
    url: https://fuel.ignitionrobotics.org
```

File: doc/ignition.in [hunk 2]
```diff
@@ -768,7 +768,7 @@ WARN_FORMAT            = "$file:$line: $text"
 # messages should be written. If left blank the output is written to standard
 # error (stderr).
 
-WARN_LOGFILE           = /tmp/ignition-fuel-tools.warn
+WARN_LOGFILE           = /tmp/gazebo-fuel-tools.warn
```
Comment (chapulina) [inline doc/ignition.in:771]:
```suggestion
WARN_LOGFILE           = /tmp/gz-fuel-tools.warn
```

**Review by chapulina — COMMENTED** [review]
Looking good to me so far, I haven't tried running it though.

I didn't check if you left a test case using `ignitionrobotics` to make sure it still works, but I think we should make sure that's tested.

File: doc/ignition.in (hunk 2 above)
Comment (nkoenig) [reply doc/ignition.in]: [7350487](https://github.com/gazebosim/gz-fuel-tools/pull/250/commits/73504877606e4229127a927022be043cdc6674e9)

**Comment by chapulina** [comment]
* Can we close this one now that #263 is in?

**Pull request closed without merging**